astropy_helpers/sphinx/ext/automodapi.py

```diff
@@ -52,6 +52,14 @@
         given, only objects that are actually in a subpackage of the package
         currently being documented are included.
 
+    * ``:inherited-members: True | False``
```

Comment:
I don't see this as documentation for `automodsumm_inherited_memebers`. As you say, that is documented elsewhere (below). If you have an idea how I can word this and propeprly explain what it does without mentioning that `automodsumm_inherited memebers` is (a) the global option (so you know what you are overriding) and (b) what it does (so you know what effect it will have when you set the override) please let me know.

I don't see how I can avoid any dublication of info about automordsumm_inherited_memeber.